base-compat/src/Data/Tuple/Compat.hs

```diff
@@ -8,8 +8,10 @@ module Data.Tuple.Compat
   , curry
   , uncurry
   , swap
-#if MIN_VERSION_ghc_prim(0,7,0)
-  , Solo(..)
+#if MIN_VERSION_ghc_prim(0,10,0)
+  , Solo(MkSolo, Solo)
+#elif MIN_VERSION_ghc_prim(0,7,0)
+  , Solo(Solo)
```

Comment:
Is these actually different?

Reply:
No, it's not. I mainly did this to make it more explicit what is being re-exported, since it wasn't obvious to me until I did some investigation.